test/e2e/framework/managed/federation.go

```diff
@@ -80,6 +80,30 @@ func SetUpFederationFixture(tl common.TestLogger, clusterCount int) *FederationF
 	return f
 }
 
+func EnableTypeConfigResource(name, namespace string, config *rest.Config, tl common.TestLogger) {
```

Comment:
Please avoid adding test-specific functionality to the framework.

Reply:
resolved